appengine-java21/bigtable/src/main/java/com/example/bigtable/BigtableHelper.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2016 Google Inc.
+ * Copyright 2024 Google LLC
```

Comment:
leave original year

Reply:
Well, I've done a global change, hard to undo one by one.